### ISSUE TITLE
Popup view does not display correctly when using CrossDissolve transition

### ISSUE THREAD

**Issue by rramirez-dev** [AvdLee/RocketSimApp#70]
**Describe the bug**
When displaying a popup view using a CrossDissolve transition the popup box appears translucent.

**To Reproduce**
1. Clone the GitHub Follower app from https://github.com/rramirez-dev/GitHub-Followers
2. Build and Launch the app in the iPhone 12 Pro Simulator
3. Start a RocketSim recording
4. Open the app and search for a GitHub user
5. Select the + sign to add the user to your favorites
6. Dismiss the popup box
7. Stop the recording and create a GIF

**Expected behavior**
The popup should not be translucnet when displayed in the GIF animation

**Screenshots**
Cross Dissolve
![PopupCrossDissolve](https://user-images.githubusercontent.com/22773854/104231745-a20c7c00-540c-11eb-92ce-54662e963ac9.gif)

Cover Vertical
![PopupCoverVerticle](https://user-images.githubusercontent.com/22773854/104231853-c5cfc200-540c-11eb-9225-5e3535e9453c.gif)

**Desktop (please complete the following information):**
 iOS - OS: [e.g. iOS]
 14.1 - Version [e.g. 22]

**Additional context**
This also occurs on other apps where I use a CrossDissolve transition to display a popup view

**Comment by AvdLee** [comment]
@rramirez-dev can you share the result of making a recording using `xcrun simctl io recordVideo booted test.mp4` from the terminal? It helps me debug this issue.

**Comment by rramirez-dev** [comment]
I get an "Invalid device: recordVideo" error

**Comment by rramirez-dev** [comment]
There were no results in the terminal

**Comment by AvdLee** [comment]
@rramirez-dev can you try again using:

```
xcrun simctl io booted recordVideo test.mp4
```

You should see:

```
Recording started
```

after which you can play around with your app. To end the recording, press ctrl+C

**Comment by rramirez-dev** [comment]
I did see "Recording Started" and it did create an test.mp4. When I said that I didn't see any output, I was talking about errors.

**Comment by AvdLee** [comment]
@rramirez-dev alright, awesome. The only thing I need is for you to share that `test.mp4` here in this thread so I can compare it to the RocketSim recording 🙂

**Comment by rramirez-dev** [comment]
Here is the test.mp4
https://user-images.githubusercontent.com/22773854/105081621-a578b680-5a4f-11eb-9c80-f409968ed00b.mp4

**Comment by AvdLee** [comment]
@rramirez-dev the video doesn't work for me 🤔  Does it work for you?

**Comment by rramirez-dev** [comment]
Yes. the video work for me.

**Comment by AvdLee** [comment]
@rramirez-dev I only get a black screen when clicking the link. Can you make it embed just like you did with your original post videos?

**Comment by rramirez-dev** [comment]
https://user-images.githubusercontent.com/22773854/105508013-19ea6a00-5c89-11eb-92dd-71d4426aa89f.mp4

**Comment by AvdLee** [comment]
![image](https://user-images.githubusercontent.com/4329185/105694865-b74fd300-5f01-11eb-9de4-29ebb808bf20.png)

The video is still not working, unfortunately.

**Comment by rramirez-dev** [comment]
The video only works in Safari. I tried tp play it in Chrome and Firefox and got the same error you did.

**Comment by AvdLee** [comment]
Alright, great that works indeed.

I miss the desktop information in the issue, which helps me to reproduce this locally.
- Which Xcode version did you use?
- Which Mac version did you use?
- Which RocketSim version did you use?

**Comment by rramirez-dev** [comment]
Xcode Version 12.3 (12C33)
Catalina Version 10.15.7
RocketSim Version 3.2.1(40)

**Comment by AvdLee** [comment]
I've improved the frame rate of GIFs which is the only thing I can do to improve for this issue. Will be released with 3.2.2
